packages/radio-group/test/lion-radio-group.test.js

```diff
@@ -11,6 +11,43 @@ import '../lion-radio.js';
 const fixture = /** @type {(arg: TemplateResult) => Promise<LionRadioGroup>} */ (_fixture);
 
 describe('<lion-radio-group>', () => {
+  it('restores to empty modelValue if changes were made', async () => {
+    const el = await fixture(html`
+      <lion-radio-group name="gender" label="Gender">
+        <lion-radio label="Male" .choiceValue=${'male'}></lion-radio>
+        <lion-radio label="Female" .choiceValue=${'female'}></lion-radio>
+      </lion-radio-group>
+    `);
+    el.formElements[0].checked = true;
+    expect(el.modelValue).to.deep.equal('male');
+
+    el.resetGroup();
+    expect(el.modelValue).to.deep.equal('');
+  });
+
+  it('restores default values if changes were made', async () => {
+    const el = await fixture(html`
+      <lion-radio-group name="gender" label="Gender">
+        <lion-radio label="Male" .choiceValue=${'male'}></lion-radio>
+        <lion-radio label="Female" .modelValue=${{ value: 'female', checked: true }}></lion-radio>
+        <lion-radio label="Other" .choiceValue=${'other'}></lion-radio>
+      </lion-radio-group>
+    `);
+    el.formElements[0].checked = true;
+    expect(el.modelValue).to.deep.equal('male');
+
+    el.resetGroup();
+    expect(el.modelValue).to.deep.equal('female');
+
+    el.formElements[2].checked = true;
```

Comment:
this section is still failing, and why i started to add all the other tests.

Reply:
Is this resolved or still failing?

Reply:
This is fixed via: https://github.com/ing-bank/lion/pull/1061#discussion_r529391587